sys-fs/lvm2/lvm2-2.02.188-r2.ebuild

```diff
@@ -123,6 +123,13 @@ src_prepare() {
 		sed -i -e 's/^\(MAN7+=.*\) $(LVMTHINMAN) \(.*\)$/\1 \2/' man/Makefile.in || die
 	fi
 
+	# Flatcar: The configure script tries to detect the systemd
+	# util dir without providing a way for us to override it, so
+	# modify the script.
+	sed -i \
+	    -e "s#^\(systemdutildir=\).*#\1$(systemd_get_utildir)#" \
+	    configure{.ac,} || die
```

Comment:
I am not sure I like the hack.
But looking at the lvm2 source tree, I think it is not possible to pass an option like `--with-systemdutildir`. 
So it is ok.

Reply:
I don't like the hack too - it's just plain terrible and ugly. But [as you saw](https://sourceware.org/git/?p=lvm2.git;a=blob;f=configure.ac;h=6cdf1a7e652a982cd648e8b194d774ebfeb9aa41;hb=HEAD#l1636), there isn't a way to override it. I could try sending a patch to lvm2 to add such an option, but I remember that Kai's patch that we still carry also wasn't acknowledged, so no high hopes here. I'll put it on my TODO list.